graph/tracer.go

```diff
@@ -61,8 +63,7 @@ func (t tracer) InterceptOperation(ctx context.Context, next graphql.OperationHa
 func (t tracer) InterceptField(ctx context.Context, next graphql.Resolver) (interface{}, error) {
 	fc := graphql.GetFieldContext(ctx)
 
-	span := opentracing.SpanFromContext(ctx)
-	span.SetOperationName(fc.Object + "_" + fc.Field.Name)
+	span, ctx := opentracing.StartSpanFromContext(ctx, fc.Object+"_"+fc.Field.Name)
```

Comment:
I'm not sure why the original code used `SpanFromContext` instead of `StartSpanFromContext`, since I'd imagine we would want new spans for these, so I'm trying it out this way

Reply:
I wonder if their implementation just will return a new one if one doesn't exist